scrapy/utils/log.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import warnings
 from logging.config import dictConfig
-
+from logging.handlers import RotatingFileHandler
 from twisted.python import log as twisted_log
```

Comment:
:lipstick: Please, keep the empty line separating imports from the standard library from imports from third-party libraries.